conda/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: "openfisca-core"
+  version: "0.0.1"
+
+source:
+  url: https://pypi.io/packages/source/O/OpenFisca-Core/OpenFisca-Core-35.7.5.tar.gz
+  #path: ..
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  # script: python setup.py install
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
```

Comment:
What happens if Python 4 is released and openfisca-core still works with Python 3.x? 🤔

Reply:
Thanks, I've added an upper limit.